src/components/flex/flex_item.tsx

```diff
@@ -61,18 +96,30 @@ export const OuiFlexItem: FunctionComponent<
 > = ({
   children,
   className,
-  grow = true,
+  grow = true, // default true -> flex-grow: 1 and flex-basis: 0%
+  shrink = null, // default null for flex-shrink
```

Comment:
Right, I think we should maintain functionality as much as possible. Since `flex-shrink` isn't specified anywhere, I think we should make it default to what it would have been previous to this change

(For clarity, this comment is referring to the `shrink` default value)

Reply:
So, when you define the default value for `shrink` as `null` (i.e.`shrink=null`), this prop does not apply any CSS property related to flex-shrink. The `OuiFlexElement` inherits what comes from the parent (`OuiFlexGroup`) or the default from the Browser (which is `flex-shrink:1`). So the default value I use work for this case, without changing the design implement for the whole element.